crypto/crypto.go

```diff
@@ -0,0 +1,36 @@
+package crypto
+
+import (
+	"golang.org/x/crypto/ed25519"
+	"golang.org/x/crypto/sha3"
+)
+
+const (
+	HashSizeByte = 32
+	HashID       = "SHAKE128"
+)
+
+func Digest(ms ...[]byte) []byte {
+	h := sha3.NewShake256()
```

Comment:
Shouldn't this be `sha3.NewShake128()`, so that it uses exactly what it says on the tin? Doesn't seem like it affects the security either way, just the amount of internal state consumed.